src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/extension/AbstractNameValueHeader/build-parameters-tooltip.js

```diff
@@ -0,0 +1,6 @@
+const buildId = document.querySelector('.pipeline-info .revision .title').dataset.buildId;
+jQuery('#build-parameters-' + buildId).tooltip({
```

Comment:
This used to be `#build-parameters-trigger-${buildGrid.get(0,0).getId()}`. I see the `buildId`, but don't see `trigger`. Has it been changed as a part of some other PR?

Reply:
Yes, I left a note in #156.

I changed it to remove the "trigger" suffix as that ID seemed to belong to the header of the build-card. Instead, now when you hover over the actual body of the card and specifically the parameters, the tooltip pops up.

Reply:
Ah, gotcha